tests/Orckestra.Composer.MyAccount.Tests/app.config

```diff
@@ -8,7 +8,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="System.Web.Mvc" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-5.2.7.0" newVersion="5.2.7.0" />
+        <bindingRedirect oldVersion="0.0.0.0-4.0.0.1" newVersion="4.0.0.1" />
```

Comment:
Could you please validate, do we need to change this to 4.0.0.1? The dependency in the project for the System.Web.MVC is 5.2.7.0